zero_bin/leader/src/main.rs

```diff
@@ -141,3 +141,17 @@ async fn main() -> Result<()> {
 
     Ok(())
 }
+
+/// Attempt to load in the local `.env` if present and set any environment
+/// variables specified inside of it.
+///
+/// To keep things simple, any IO error we will treat as the file not existing
+/// and continue moving on without the `env` variables set.
+fn load_dotenvy_vars_if_present() -> anyhow::Result<()> {
+    match dotenv() {
+        Ok(_) | Err(dotenvy::Error::Io(io::Error { .. })) => Ok(()),
+        _ => Err(anyhow!(
+            "Found local `.env` file but was unable to parse it!"
+        )),
+    }
```

Comment:
This is the only thing that I'm not sure about. Is it ok to keep it simple and treat any IO error as the file missing? This still means that a parsing error of `.env` will still crash (which I think is the best behavior here?).

Reply:
@BGluth Just put a warning message that .env file could not be parsed and that program is skipping it

Reply:
Ok sounds good!